doc/src/manual/parallel-computing.md

```diff
@@ -1699,6 +1699,15 @@ requirements for the inbuilt `LocalManager` and `SSHManager`:
     authenticated via public key infrastructure (PKI). Authentication credentials can be supplied
     via `sshflags`, for example ```sshflags=`-i <keyfile>` ```.
 
+    Internally, SSH multiplexing is used to create a tunnel between the master and workers. If you have
+    been configured SSH multiplexing on your own and the connection has already been established,
```

Comment:
Typo: should probably be "have configured" instead of "have been configured"?

Reply:
Right. Fixed in 79cf3d74e58a755a4aaae9d6ccbbcc6d51a46393